parse_roms.py

```diff
@@ -95,15 +97,13 @@ def generate_rom_entries(self, name: str, roms: [ROM], rom_prefix: str, save_pre
         return ROM_ENTRIES_TEMPLATE.format(name=name, body=body, rom_count=len(roms))
 
 
-    def generate_char_array(self, name: str, ROM: ROM) -> str:
-        rom_data = ROM.data()
-        template = "const uint8_t {name}[]  __attribute__((section (\".extflash_game_rom\"))) = {{\n"
-        char_array = template.format(name=name)
-        char_array += ", ".join([hex(x) for x in rom_data])
-        char_array += "};\n"
-
-        return char_array
-
+    def generate_object_file(self, ROM: ROM) -> str:
+        # convert rom to an .o file and place the data in the .extflash_game_rom section
+        subprocess.run([ "arm-none-eabi-objcopy", "--rename-section", ".data=.extflash_game_rom,alloc,load,readonly,data,contents", \
```

Comment:
Could you add so that these subprocess invocations also take `GCC_PATH` into consideration?